### DISCUSSION TITLE
Disambiguation between errors and warnings when loading models

### DISCUSSION THREAD

**Discussion by stillonearth** [google-deepmind/mujoco#591] [General]
In some cases MuJoCo will throw an error such

```Error loading model: Degenerate triangle found in mesh `trunk`, its orientation will not be checked.```

I'm getting this from loading https://github.com/deepmind/mujoco_menagerie/blob/main/unitree_a1/a1.xml

Is there a way to distinguish fatal errors and warning programmatically with `mj_loadXML`?

**Comment by kevinzakka** [comment]
Note the degenerate triangle warning will be removed in the next release, see da33c5922fa229de51d70bf9a4d20e5856d075d6.
